tests/test_engine.py

```diff
@@ -2243,3 +2244,26 @@ def parent():
         " the same flow."
         in caplog.text
     )
+
+
+@patch(
+    "prefect.utilities.collections.visit_collection",
+    wraps=prefect.utilities.collections.visit_collection,
```

Comment:
How have I been doing this for `[redacted]` years and never saw that `mock.patch` takes this `wraps` argument?!  Very tidy